autogpt_platform/backend/backend/api/features/experts/scheduling.py

```
@@ -0,0 +1,285 @@
+    orphaned webhook firing; the run-time gate (``enforce_expert_run_budget``
+    refusing archived/paused experts) is the backstop for anything missed.
+    """
+    await prisma.models.AgentPreset.prisma().update_many(
+        where={"expertId": expert_id, "userId": user_id, "isDeleted": False},
+        data={"isActive": False},
+    )
+    scheduler = get_scheduler_client()
+    for schedule in await _get_expert_schedules(user_id, expert_id):
+        try:
+            await scheduler.delete_schedule(schedule.id, user_id=user_id)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)